### DISCUSSION TITLE
Community Updates

### DISCUSSION THREAD

**Discussion by dysfunc** [sveltestrap/sveltestrap#89] [Announcements]
**Issues/Tickets**
I promise we're not ignoring any of the recent issues coming in. I've had several life events happen this year, including a newborn, so I'm taking care of things for the next couple of weeks and then I'll be back to address all of outstanding items.

**Svelte 5 Support**
We are planning to support Svelte 5. This will require a decent amount of changes to the codebase, BUT it will kick-off once I return. I need to handle the update for my own production apps - so it is a priority.

**Maintainers**
The original Sveltestrap owner @bestguy decided to join my company 👏🏻 so he'll be helping to maintain this library alongside @msmauric.

**Comment by vince-winkintel** [comment]
Any update on when there will be Svelte 5 support?

**Comment by mattmiller1996** [comment]
I am also looking for an update with this! Thanks.
